src/pages/progress_reports/DarCloseout.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import { FormField, FormFieldTypes } from '../../components/forms/forms';
+
+interface FormStateInterface {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    [key: string]: any;
+}
+
+interface DarCloseoutProps {
+    readonly state: string;
+}
+
+interface FormFieldChange {
+    key: string;
+    value: boolean | string;
+}
+
+export default function DarCloseout(_props: DarCloseoutProps): React.JSX.Element {
+    const [formState, setFormState] = useState<FormStateInterface>({});
+
+    return (
+        <div data-cy='dar-closeout'>
+            <div className='progress-report-step-card'>
+                <h2>Step 4: DAR Closeout</h2>
+
+                <div className='progress-report-row'>
+                    <div>
+                        <h3>4.1 Closeouts</h3>
+                        <p>
+                            If you are ready to finish work on this project, please complete this DAR Closeout section.
+                            <br /><br />
+                            By completing this page, upon project close-out, the PI and all approved users agree to destroy all copies, versions, and derivations of the dataset(s) retrieved from NIH-designated controlled-access databases, on both local servers and hardware, and if cloud computing was used, delete the data and cloud images from cloud computing provider storage, virtual machines, databases, and random access archives, except as required by publication practices, institutional policies, or law to retain them.
+                            <br /><br />
+                            This close-out will be submitted to the Signing Official (SO) of the project for approval. The SO is required to confirm the data destruction and insure retained data is encrypted, properly stored, and deleted at the appropriate time to comply with data security policies. If approved by the SO, the request will be sent to the Data Access Committee (DAC) for final approval. The project will be closed out after DAC approval is completed.
+                        </p>
+                    </div>
+                    <div style={{ marginTop: '20px' }}>
+                        <h4>Reasons for Project Closeout</h4>
+                        <FormField
+                            id='closeoutCompleted'
+                            type={FormFieldTypes.CHECKBOX}
+                            toggleText='The Requestor has completed his/her project'
```

Comment:
Could we have this say 'The Requestor has completed their project'? Or do we have to use the his/her language?

Reply:
Not sure if this is NIH compliant language! Let's ask Jonathan/Victoria.